Library/Formula/commandbox-be.rb

```diff
@@ -0,0 +1,29 @@
+class CommandboxBe < Formula
+  desc "A CLI, package manager, app scaffolding tool with embedded CFML server"
+  homepage "http://www.ortussolutions.com/products/commandbox"
+  url "http://integration.stg.ortussolutions.com/artifacts/ortussolutions/commandbox/3.1.0/commandbox-bin-3.1.0.zip"
+  sha1 "2e3bab8443b7965931cd7c2f4edefc14c6e4297c"
+  
+  depends_on :arch => :x86_64
+  depends_on :java => "1.7+"
+
+  resource 'apidocs' do
+    url "http://integration.stg.ortussolutions.com/artifacts/ortussolutions/commandbox/3.1.0/commandbox-apidocs-3.1.0.zip"
+    sha1 "5b20d54a3c24f16fb18cc93d88bd6df1fc317be2"
+  end
+
+  def install
+    bin.install 'box'
```

Comment:
Just curious, how big of a deal is the name?  We've been using `box` as the binary name for over two years now and it's important to have a short name so the CLI commands aren't long.  No other mainstream software seems to be using that binary name, and certainly not in the CFML or Java world which would be the majority of installations.  We also have a lot of docs and guides using the `box` binary name too, so it would be a little weird for our debian, rpm, windows, and binary installs to all be "box", but for Brew to install something else.